cmd/contour/servecontext.go

```diff
@@ -208,6 +209,12 @@ type TLSConfig struct {
 	FallbackCertificate FallbackCertificate `yaml:"fallback-certificate,omitempty"`
 }
 
+type ServerConfig struct {
+	// Defines the XDSServer to use for `contour serve`
+	// Defaults to "contour"
+	XDSServerType string `yaml:"xds-server-type,omitempty"`
```

Comment:
nit: I slightly prefer the camelcase versions of config key names so, `xdsServerType`, over the hyphenated one., but I didn't say anything last time, so I'm okay with leaving it be.

Reply:
I was just following how much of everything else is setup which is the hyphenated one.